config/settings.yml

```diff
@@ -390,3 +392,14 @@ bing:
 flipper:
   username: SUPER_SECRET_USERNAME
   password: SUPER_SECRET_PASSWORD
+
+oit_lighthouse2:
+  sql52:
+    hostname: vhacdwdwhsql52.vha.med.va.gov
```

Comment:
Should we take this out? They're not actually being used, and I don't think they will be